GenderPayGap.WebUI/BackgroundJobs/Jobs/PurgeOrganisationsJob.cs

```diff
@@ -9,12 +9,10 @@
 using GenderPayGap.Database;
 using GenderPayGap.Extensions;
 using GenderPayGap.Extensions.AspNetCore;
-using Microsoft.Azure.WebJobs;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 
-namespace GenderPayGap.WebJob
+namespace GenderPayGap.WebUI.BackgroundJobs.Jobs

```

Comment:
Sweet I hadn't realised this would all live in WebUI. Does that mean we can get rid of another project?

Reply:
Yes, soon we'll be getting rid of the WebJob project completely :-)

Reply:
🎉 🎉 🎉